dev/a11y_assessments/lib/use_cases/tab_bar.dart

```diff
@@ -0,0 +1,66 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'use_cases.dart';
+
+class TabBarUseCase extends UseCase {
```

Comment:
This should probably call TabBarViewUseCase

Reply:
`TabBar` widget  is wrapped in  AppBar bottom. it's the widget i want to test. 

`TabBarView` is used in Scaffold body  and it's to show the switching result of the tab bar

Reply:
I think we should consider them as one component since they are mostly use together. so the name should try to catch both widgets.

Reply:
renamed